docs/docs/dev_docs/testing/cheat_codes.md

```diff
@@ -0,0 +1,449 @@
+---
+title: Cheat Codes
+---
+
+## Introduction
+
+Most of the time, simply testing your smart contract isn't enough. To manipulate the state of the Aztec blockchain, as well as test for specific reverts and events, the sandbox is shipped with a set of cheatcodes.
+
+Cheatcodes allow you to change the time of the L2 block, load certain state or more easily manipulate L1 instead of having to write dedicated RPC calls to anvil. 
+
+:::info Prerequisites
+If you aren't familiar with [Anvil](https://book.getfoundry.sh/anvil/), we recommend reading up on that since Aztec Sandbox uses Anvil as the local L1 instance.
+:::
+
+### Aims
+
+The guide will cover how to manipulate the state of the:
+- L1 blockchain;
+- Aztec network.
+### Why is this useful?
+
+<!-- Contextualise why a dapp developer needs this. What use cases / products / features does this unlock? Any real world examples? -->
+
+### Dependencies
+
+For this guide, the following Aztec packages are used:
+- @aztec/aztec.js
+
+### Initialisation
+```js
+import { createAztecRpcClient, CheatCodes } from '@aztec/aztec.js';
+const aztecRpcUrl = 'http://localhost:8080';
```

Comment:
The create signature is 
create(ethRpcUrl, aztecRpcUrl)

Reply:
Doesn't look like it?
For creating the aztec rpc client, the sandbox uses `createAztecRpcClient(aztecRpcUrl);` [here](https://github.com/AztecProtocol/aztec-packages/blob/4af4845f80b1be548efa1ca79f5588bb1c7f1423/yarn-project/aztec-sandbox/src/examples/uniswap_trade_on_l1_from_l2.ts#L58)

And for cheatcodes it is `create(rpcUrl: string, aztecRpc: AztecRPC)`

Reply:
My mistake, youre absolutely correct